src/sass/layouts/_hero.scss

```
@@ -0,0 +1,40 @@
+.hero__container {
+  background-color: var(--hero-bg);
+}
+
+.hero__upertitle {
+  font-family: 'Titan One';
+  font-style: normal;
+  font-weight: 400;
```

це можна не писати є у стандартних налаштуваннях браузера
```suggestion
  font-style: normal;
  font-weight: 400;
```